test/integration/requests/fee.ts

```diff
@@ -1,26 +1,83 @@
 import { assert } from 'chai'
 import _ from 'lodash'
 
-import { FeeRequest } from '../../../src'
+import { FeeRequest } from 'xrpl-local'
+
 import serverUrl from '../serverUrl'
 import { setupClient, suiteClientSetup, teardownClient } from '../setup'
 
 // how long before each test case times out
 const TIMEOUT = 20000
 
-describe('Server Info Fee', function () {
+describe('fee', function () {
   this.timeout(TIMEOUT)
 
   before(suiteClientSetup)
   beforeEach(_.partial(setupClient, serverUrl))
   afterEach(teardownClient)
 
-  it('fee', async function () {
+  it('base', async function () {
     const request: FeeRequest = {
       command: 'fee',
     }
     const response = await this.client.request(request)
-    assert.equal(response.status, 'success')
-    assert.equal(response.type, 'response')
+    const expected = {
+      id: 5,
```

Comment:
nit:
```suggestion
      id: 0,
```
To follow what the fixtures do. Apply to the rest of the files as well.